plugins/controlboard/include/ControlBoard.hh

```diff
@@ -59,14 +60,36 @@ private:
     yarp::os::Network m_yarpNetwork;
     yarp::os::Property m_pluginParameters;
 
+    enum class UnitsTypeEnum
+    {
+        METRIC = 0,
```

Comment:
What do you mean by "METRIC" ? How it is different from "SI" ? I thought they are equivalent?

Reply:
This is a distinction ported from the original plugin, see the [setPIDsForGroup_POSITION](https://github.com/robotology/gazebo-yarp-plugins/blob/cb3521f7543a5a07e2a65a82dedd2eb2ec2cea12/plugins/controlboard/src/ControlBoardDriver.cpp#L1056-L1201) method. I think this property only means degrees vs. radians, since the only place where it is used is in that method where, based on that configuration parameter, the PID gains are converted from degrees to radians or not.

Reply:
Ahh, I see. The related code is https://github.com/robotology/gazebo-yarp-plugins/blob/cb3521f7543a5a07e2a65a82dedd2eb2ec2cea12/plugins/controlboard/src/ControlBoardDriver.cpp#L1174-L1185 . Do you think this "metric" used to identify "degrees for angles" make sense? Can you check? If it does not make sense, probably we can use another word for meaning "degrees for angles"? We can still temporary support `metric` in the config files to simplify the Gazebo Classic --> gz-sim migration, but at least for the internal enum I would try to stick to something that make sense at least for us.

Reply:
To me, "metric" is just the ancestor of the International System of Units (see https://en.wikipedia.org/wiki/International_System_of_Units), so they are basically synonyms.

Reply:
Yes I agree entirely with your considerations; sorry for not pointing out this ambiguity while porting it. I thought the same as you, that metric and SI basically mean the same thing. Maybe for now we can change in the code the naming of variables to distinguish radian from degree angles, while maintaining the same configuration files keywords.

Reply:
I confirm that the `metric_units` configuration property is used in the three methods to set the PIDs for position, velocity and impedance controls:
- position: https://github.com/robotology/gazebo-yarp-plugins/blob/cb3521f7543a5a07e2a65a82dedd2eb2ec2cea12/plugins/controlboard/src/ControlBoardDriver.cpp#L1056-L1201
- velocity:https://github.com/robotology/gazebo-yarp-plugins/blob/cb3521f7543a5a07e2a65a82dedd2eb2ec2cea12/plugins/controlboard/src/ControlBoardDriver.cpp#L1202-L1313
- impedance: https://github.com/robotology/gazebo-yarp-plugins/blob/cb3521f7543a5a07e2a65a82dedd2eb2ec2cea12/plugins/controlboard/src/ControlBoardDriver.cpp#L1314-L1397

Reply:
Done in https://github.com/robotology/gz-sim-yarp-plugins/pull/92/commits/751fdd296a5b5e5fd4194b8654f430b566407240